README.md

```diff
@@ -66,7 +66,8 @@ Follow these steps to make changes and release a new binary.
    The SHAs can be obtained by running the command `python parse-image-sha.py <TAG>`
    This will return the SHAs for kube-dns as well as node-cache images. Node-cache images are always promoted, kube-dns images are promoted if there is a change to kubedns/vulnerability fix.
 6. Images will be available in the repo registry.k8s.io/dns/. The node-cache image with tag 1.15.14 can be found at registry.k8s.io/dns/k8s-dns-node-cache:1.15.14. Older versions are at registry.k8s.io/k8s-dns-node-cache:<TAG>
-7. Submit a PR for the kubernetes/kubernetes repository to switch to the new
+7. Verify the test grid at https://testgrid.k8s.io/sig-network-gce#gci-gce-serial-kube-dns for regressions caused by the new image.
```

Comment:
This is a chicken an egg problem, you need to test it before submitting it, we can add a presubmit job optional that we can trigger on demand for this https://github.com/kubernetes/test-infra/blob/6cbed8179723b665fd8ad8a8af9a986d4a9f4596/config/jobs/kubernetes/sig-network/sig-network-misc.yaml#L790,

Reply:
I see, so we couldn't detect it before merging with the current tooling. I'll reword the instruction then.

Reply:
Done.